test/parallel/test-http-response-status-message.js

```diff
@@ -41,7 +56,7 @@ var server = net.createServer(function(connection) {
 var runTest = function(testCaseIndex) {
   var testCase = testCases[testCaseIndex];
 
-  http.get({ port: common.PORT, path: testCase.path }, function(response) {
+  http.get({port: common.PORT, path: testCase.path}, function(response) {
```

Comment:
I'm still not exactly "ok" with the churn created by changing these spacing ones. :-1: (Plus no one seems to agree on which style anyways)